applications/VeBlocks_NodeAccess_2023.md

```diff
@@ -0,0 +1,81 @@
+# VeChain General Grant Application Template
+
+## Project Overview
+
+- Project: VeBlocks free access to public Vechain blockchain nodes
+- Team Name: MiRei - I am happy to provide a detailed resume via email but not happy to have it on github.
+- Payment Address: 0xcb0c81cf6742c2ea97cec01f95c08d22045edbfc USDT (ERC 20)
```

Comment:
Our payment method has been updated to “USDC”. We kindly request you to verify this change.